internal/cmd/connect/command_custom_plugin_delete.go

```diff
@@ -0,0 +1,52 @@
+package connect
+
+import (
+	"fmt"
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/resource"
+	"github.com/spf13/cobra"
+)
+
+func (c *customPluginCommand) newDeleteCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "delete <id>",
+		Short:       "Delete a custom connector plugin.",
+		Args:        cobra.ExactArgs(1),
+		RunE:        c.delete,
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Delete a custom connector plugin",
+				Code: "confluent connect custom-plugin delete ccp-123456",
+			},
+		),
+	}
+
+	pcmd.AddForceFlag(cmd)
+	pcmd.AddContextFlag(cmd, c.CLICommand)
+
+	return cmd
+}
+
+func (c *customPluginCommand) delete(cmd *cobra.Command, args []string) error {
```

Comment:
Please take a look at other `delete` commands. I believe you need to define an "existence function" now. cc @sgagniere

Reply:
Looks like we doubled up on the feedback in some parts; I mentioned the README above.